src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -89,8 +89,8 @@ public static ImagePixelExtractingEstimator ExtractPixels(this TransformsCatalog
         /// <seealso cref= "ImageLoadingEstimator" />
         /// </remarks >
         /// <param name="catalog">The transform's catalog.</param>
-        /// <param name="inputColumnName">Name of the input column.</param>
-        /// <param name="outputColumnName">Name of the resulting output column.</param>
+        /// <param name="inputColumnName">The name of the input column.</param>
+        /// <param name="outputColumnName">The name of the resulting output column.</param>
```

Comment:
I would prefer standardize language we use for them:
```
/// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
/// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
        
```

Reply:
Added the changes in the last update of the PR